### PR TITLE
Bump L-2 versions for WP and WC

### PR DESCRIPTION
**Pull request by diegocurbelo**
Fixes #

## Changes proposed in this Pull Request:

This PR bumps the supported versions of WordPress and WooCommerce to their latest - 2 versions:

- PHP: `7.3`, `7.4` and `8.0`
- WordPress: `6.0`, `5.9` and `5.8`
- WooCommerce: `6.8.2`, `6.9.4` and `7.0.0`

**Notes:**
- "phpunit/phpunit": "7.5.20" is still needed by WordPress 5.8

## Testing instructions
Verify that ALL the `checks` for the PR are successfully completed

Screenshot 2022-11-01 at 13 42 24

---

-   [x] Covered with tests (or have a good reason not to test in description ☝️)
-   [x] Added changelog entry **in both** `changelog.txt` and `readme.txt` (or does not apply)
-   [x] Tested on mobile (or does not apply)

**Post merge**

-   [x] Added testing instructions to the [Release Testing Instructions wiki page](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Release-Testing-Instructions) (or does not apply)

### CODE REVIEW

File: .github/workflows/compatibility.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         woocommerce: [ 'beta' ]
         wordpress:   [ 'latest' ]
-        php:         [ '7.2', '7.4', '8.0' ]
+        php:         [ '7.3', '7.4', '8.0' ]
```
Comment (cesarcosta99) [inline .github/workflows/compatibility.yml:14]: Can we start supporting PHP 8.1? See latest [PHP releases](https://www.php.net/releases/index.php).

File: tests/phpunit/setup.php [hunk 2]
```diff
@@ -4,7 +4,3 @@
  */
 update_option( 'woocommerce_default_country', 'US:CA' );
 
-if ( version_compare( WC_VERSION, '6.4.1', '=' ) ) {
-	\Automattic\WooCommerce\Internal\Admin\Install::create_tables();
-	\Automattic\WooCommerce\Internal\Admin\Install::create_events();
-}
```
Comment (cesarcosta99) [inline tests/phpunit/setup.php:10]: Good catch!

**Review by cesarcosta99 — CHANGES_REQUESTED** [review]
LGTM, left a question about the supported PHP versions.

File: .github/workflows/compatibility.yml (hunk 1 above)
Comment (diegocurbelo) [reply .github/workflows/compatibility.yml:14]: Unfortunately the tests fail with PHP 8.1, we have the issue #2411 to track it.

**Review by cesarcosta99 — APPROVED** [review]
LGTM 👍

**Pull request merged**